server/redisstate.go

```diff
@@ -29,6 +29,10 @@ var RedisExpiryNonceFixForAccount = time.Duration(24 * time.Hour)
 var RedisPrefixSenderOfTxHash = RedisPrefix + "txsender-of-txhash:"
 var RedisExpirySenderOfTxHash = time.Duration(24 * time.Hour) // 1 day
 
+// Remember nonce of pending user tx
+var RedisPrefixSenderMaxNonce = RedisPrefix + "txsender-pending-max-nonce:"
+var RedisExpirySenderMaxNonce = time.Duration(24 * time.Hour) // 1 day
```

Comment:
1 day might be too long? Shouldn't these expire within 25 blocks so roughly 10-20 minutes?

Reply:
Perhaps one or two hours would be best, because it needs to take into account nonces from tx in the public mempool too